tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Utils/BearerTokenBuilder.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.IdentityModel.Tokens;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Utils
+{
+    public sealed class BearerTokenBuilder
+    {
+        private X509Certificate2 _signingCertificate;
+        private string _issuer = "http://host.docker.internal:8070/identity";
```

Comment:
Could this be any value? Or does it have to match what is defined in the Ordering.Api? Just thinking we have moved most of the Urls into config, do you think this could be another addition?

Reply:
excellent question! I'll test, and if it has to be a specific value will move it into config